### PR TITLE
Doc running

### PR DESCRIPTION
**Pull request by MatthewKhouzam**

### CODE REVIEW

**Review by bhufmann — CHANGES_REQUESTED** [review]
Please rebase this PR on latest in main and re-submit. The pom changes are already in main.

File: README.md [hunk 1]
```diff
@@ -120,3 +120,32 @@ The design philosophy of this class is heavily inspired by the trace event forma
...
+```console
+user@host:~$ javac -cp trace-event-logger-x.y.z.jar my-project
+```
+	
+`-cp`: loads the jar
+
+and then, run it as 
+
+```console
+user@host:~$ java -Djava.util.logging.config.file=path/to/logging.properties -p bin:path/to/trace-event-logger-x.y.z.jar -m pacakge/path/to.Main
```
Comment (arfio) [inline README.md:136]: pacakge -> package

File: README.md [hunk 2]
```diff
@@ -120,3 +120,32 @@ The design philosophy of this class is heavily inspired by the trace event forma
...
+
+`-p`: the packages, the location of the classes and the external logging jar
+
+`-m`: is the location of the main module to run.
+
+## Handlers
+
+There are two new `handler`s introduced.
+
+* The AsyncFileHandler: `org.eclipse.tracecompass.trace_event_logger.AsyncFileHandler` in the logging properties. It handles serialization and writing to disk in a separate thread from the the caller. Note: this will require explicitly killing a process when it exits as it is a separate thread.
```
Comment (arfio) [inline README.md:149]: from the the caller -> from the caller

File: README.md [hunk 3]
```diff
@@ -120,3 +120,32 @@ The design philosophy of this class is heavily inspired by the trace event forma
...
+
+* The SnapshotHandler: `org.eclipse.tracecompass.trace_event_logger.SnapshotHandler` in the logging properties. It write all events to a fixed length queue (culling at the front when full) and dumps the entire queue to disk when a defined latency is hit, for example when a span lasts over 30 seconds.
```
Comment (arfio) [inline README.md:151]: It write -> It writes

**Review by bhufmann — DISMISSED** [review]
LGTM. Thanks!

File: README.md [hunk 4]
```diff
@@ -120,3 +120,32 @@ The design philosophy of this class is heavily inspired by the trace event forma
...
+
+```console
+user@host:~$ java -Djava.util.logging.config.file=path/to/logging.properties -p bin:path/to/trace-event-logger-x.y.z.jar -m pacakge/path/to.Main
+```
+
+`-Djava.util.logging.config.file`: the path of the `logging.properties`
+
+`-p`: the packages, the location of the classes and the external logging jar
+
+`-m`: is the location of the main module to run.
```
Comment (arfio) [inline README.md:143]: Why not `the location of the main module to run` as the other parameters are described declaratively ?

**Review by arfio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MatthewKhouzam) [reply README.md:136]: thanks!

File: README.md (hunk 4 above)
Comment (MatthewKhouzam) [reply README.md:143]: done

File: README.md (hunk 3 above)
Comment (MatthewKhouzam) [reply README.md:151]: done!

File: README.md (hunk 2 above)
Comment (MatthewKhouzam) [reply README.md:149]: done!

**Comment by MatthewKhouzam** [comment]
I think they are done! please confirm @arfio  (after timezone appropriate meal.)

**Review by arfio — APPROVED** [review]
LGTM ! thank you

**Pull request merged**